hack/tools/release/notes.go

```diff
@@ -512,12 +512,15 @@ func generateReleaseNoteEntry(c *commit) (*releaseNoteEntry, error) {
 		if strings.Contains(entry.title, "CAEP") || strings.Contains(entry.title, "proposal") {
 			entry.section = proposals
 		}
-	case strings.HasPrefix(entry.title, ":seedling:"), strings.HasPrefix(entry.title, "🌱"):
-		entry.section = other
-		entry.title = removePrefixes(entry.title, []string{":seedling:", "🌱"})
 	case strings.HasPrefix(entry.title, ":warning:"), strings.HasPrefix(entry.title, "⚠️"):
 		entry.section = warning
 		entry.title = removePrefixes(entry.title, []string{":warning:", "⚠️"})
+	case strings.HasPrefix(entry.title, "🚀"), strings.HasPrefix(entry.title, "🌱 Release v1."):
```

Comment:
Can we add a TODO to remove this once it's no longer relevant? If I understand right it should be unnecessary after the coming releases for 1.4.X and 1.5.X